packages/server-wallet/src/protocols/__test__/application.test.ts

```diff
@@ -16,13 +16,13 @@ const outcome2 = simpleEthAllocation([
   {amount: BN.from(5), destination: bob().destination},
   {amount: BN.from(5), destination: alice().destination},
 ]);
-const prefundState = {outcome, turnNum: 0};
-const prefundState2 = {outcome: outcome2, turnNum: 0};
-const postFundState = {outcome, turnNum: 2};
-const closingState = {outcome, turnNum: 4, isFinal: true};
+const prefundState = {outcome, turnNum: 0, participants: [alice(), bob()]};
+const prefundState2 = {outcome: outcome2, turnNum: 0, participants: [alice(), bob()]};
+const postFundState = {outcome, turnNum: 2, participants: [alice(), bob()]};
+const closingState = {outcome, turnNum: 4, isFinal: true, participants: [alice(), bob()]};
```

Comment:
This is probably more readable?
```suggestion
const participants = [alice(), bob()];
const prefundState = {outcome, turnNum: 0, participants};
const prefundState2 = {outcome: outcome2, turnNum: 0, participants};
const postFundState = {outcome, turnNum: 2, participants};
const closingState = {outcome, turnNum: 4, isFinal: true, participants};
```